wgsl/index.bs

```diff
@@ -2463,6 +2545,10 @@ storage_texture_type
   | TEXTURE_STORAGE_2D_ARRAY
   | TEXTURE_STORAGE_3D
 
+storage_texture_access
```

Comment:
Suggest just reusing the access mode grammar between variables and storage textures. The storage texture rules prohibit read_write. It's ok if the grammar accepts more than is strictly valid.

Reply:
I'm fine with that.  Clearly I initially leaned the other way: be more specific. But there's already significant validation required for other reasons.